src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1696,4 +1696,7 @@ This version used in: {2}</value>
   <data name="Add_null_checks_for_all_parameters" xml:space="preserve">
     <value>Add null checks for all parameters</value>
   </data>
+  <data name="Add_DebuggerDisplay" xml:space="preserve">
+    <value>Add DebuggerDisplay</value>
```

Comment:
probably nicer to have as either `Add DebuggerDisplay attribute` or as lang-specific strings like `Add [DebuggerDisplay]`.

Reply:
What do you think of `Add debugger display`? That is the terminology of what I want versus how I achieve it.

Reply:
I actually like having bits of 'syntax' in the title.  It helps make it clearer (imo) as to what teh feature will do.  but I don't care much.  I would prefer `Add 'DebuggerDisplay'` probably the most or `Add 'DebuggerDisplay' attribute` if you really did just want a single title for both languages.